### PR TITLE
Minor addition to Irreversible block definition

### PR DESCRIPTION
**Pull request by bobgt**
Appended text to the current definition of **Irreversible Block**

### CODE REVIEW

**Comment by iamveritas** [comment]
something went wrong here, I think the `bobgt-patch-4` branch was not created of off `release/2.0.x` branch.

**Review by iamveritas — CHANGES_REQUESTED** [review]
something went wrong here, I think the bobgt-patch-4 branch was not created of off release/2.0.x branch.

File: glossary.yaml [hunk 1]
```diff
@@ -99,7 +99,7 @@ terms:
     name: "Private Network"
   - definition: "A blockchain application is a software application that has integrated a blockchain in its architecture as the storage layer for part of, or, all of its data. This includes software applications that do not own their own contract on the blockchain and instead only interact with the system contracts of the blockchain."
     name: "Blockchain Application"
-  - definition: "A block is considered irreversible (i.e. immutable) on an EOSIO based blockchain when 2/3rd of the currently elected block producers have acknowledged it."
+  - definition: "Irreversible blocks are blocks that contain confirmed, final transactions. A block is considered irreversible (i.e. immutable) on an EOSIO-based blockchain when 2/3rd of the currently elected block producers acknowledge it."
```
Comment (lparisc) [inline glossary.yaml:102]: correction: it is 2/3rds plus one
suggestion: use term "supermajority" in this context
consider:
when 2/3rd of the currently elected block producers acknowledge it. --> when a supermajority consisting of 2/3rds plus one of the currently elected block producers have confirmed the block.

**Review by lparisc — CHANGES_REQUESTED** [review]

**Comment by lparisc** [comment]
> something went wrong here, I think the bobgt-patch-4 branch was not created of off release/2.0.x branch.

Just rebased to `develop` and it's fine now. However, 200+ changed files is not a good sign, regardless. The two branches should not be that far off. Seems like some PRs have not been properly backported or up-ported and now it's a mess.

**Comment by bobgt** [comment]
> something went wrong here, I think the bobgt-patch-4 branch was not created of off release/2.0.x branch.

Err, I think I edited the file by clicking the **Edit** button on the Dev Portal page. That's why it created the bobgt-patch-4 branch from `develop`?

File: glossary.yaml (hunk 1 above)
Comment (bobgt) [reply glossary.yaml]: Suggestion applied. I liked the word **_supermajority_**.

**Comment by lparisc** [comment]
> > something went wrong here, I think the bobgt-patch-4 branch was not created of off release/2.0.x branch.
> 
> Err, I think I edited the file by clicking the **Edit** button on the Dev Portal page. That's why it created the bobgt-patch-4 branch from `develop`?

Yes, since the edit feature on the dev portal picks `develop` by default. It's alright since it's still applicable to both branches. Just cherry-pick all commits to `release/2.0.x` in another PR once this one is approved.

**Comment by bobgt** [comment]
> > something went wrong here, I think the bobgt-patch-4 branch was not created of off release/2.0.x branch.
> 
> Just rebased to `develop` and it's fine now. However, 200+ changed files is not a good sign, regardless. The two branches should not be that far off. Seems like some PRs have not been properly backported or up-ported and now it's a mess.

Thanks for rebasing. 

What steps do we need to take to ensure that the PRs are backported and up-ported?

File: glossary.yaml [hunk 2]
```diff
@@ -99,7 +99,7 @@ terms:
     name: "Private Network"
   - definition: "A blockchain application is a software application that has integrated a blockchain in its architecture as the storage layer for part of, or, all of its data. This includes software applications that do not own their own contract on the blockchain and instead only interact with the system contracts of the blockchain."
     name: "Blockchain Application"
-  - definition: "A block is considered irreversible (i.e. immutable) on an EOSIO based blockchain when 2/3rd of the currently elected block producers have acknowledged it."
+  - definition: "Irreversible blocks are blocks that contain confirmed, final transactions. A block is considered irreversible (i.e., immutable) on an EOSIO-based blockchain when a supermajority, consisting of 2/3rds, plus one of the currently elected block producers have confirmed the block."
```
Comment (lparisc) [inline glossary.yaml:102]: the supermajority is 2/3rds plus one, not 2/3rds.
supermajority, consisting of 2/3rds, plus one of --> supermajority, consisting of 2/3rds plus one, of

Comment (bobgt) [reply glossary.yaml]: Clear now?

**Review by lparisc — APPROVED** [review]

**Comment by lparisc** [comment]
> > > something went wrong here, I think the bobgt-patch-4 branch was not created of off release/2.0.x branch.
> > 
> > 
> > Just rebased to `develop` and it's fine now. However, 200+ changed files is not a good sign, regardless. The two branches should not be that far off. Seems like some PRs have not been properly backported or up-ported and now it's a mess.
> 
> Thanks for rebasing.
> 
> What steps do we need to take to ensure that the PRs are backported and up-ported?

For a PR merged to `develop`, if the changes are also applicable to a `release` branch, the commits must be backported to that `release` branch.

For a PR merged to a given `release` branch, if the changes are also applicable to a lower-versioned `release` branch, the commits must be backported to that branch. If the changes are also applicable to a higher-versioned `release` branch or the `develop` branch, the commits must be up-ported to those branches.

For either backporting or up-porting, the process involves cherry-picking the applicable commits. Check the "PR Resolution" process in Confluence.

**Comment by bobgt** [comment]
> > > > something went wrong here, I think the bobgt-patch-4 branch was not created of off release/2.0.x branch.
> > > 
> > > 
> > > Just rebased to `develop` and it's fine now. However, 200+ changed files is not a good sign, regardless. The two branches should not be that far off. Seems like some PRs have not been properly backported or up-ported and now it's a mess.
> > 
> > 
> > Thanks for rebasing.
> > What steps do we need to take to ensure that the PRs are backported and up-ported?
> 
> For a PR merged to `develop`, if the changes are also applicable to a `release` branch, the commits must be backported to that `release` branch.
> 
> For a PR merged to a given `release` branch, if the changes are also applicable to a lower-versioned `release` branch, the commits must be backported to that branch. If the changes are also applicable to a higher-versioned `release` branch or the `develop` branch, the commits must be up-ported to those branches.
> 
> For either backporting or up-porting, the process involves cherry-picking the applicable commits. Check the "PR Resolution" process in Confluence.

This is clear and helpful! Thanks!

**Review by iamveritas — APPROVED** [review]

**Review by iamveritas — APPROVED** [review]

**Pull request merged**